app/test_main.py

```diff
@@ -1,3 +1,29 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_check_human_age_when_animal_age_less_than_15() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_check_human_age_when_animal_age_equal_15() -> None:
+    assert get_human_age(15, 15) == [1, 1]
+
+
+def test_check_human_age_when_animal_age_less_than_24() -> None:
+    assert get_human_age(23, 23) == [1, 1]
+
+
+def test_check_human_age_when_animal_age_equal_24() -> None:
+    assert get_human_age(24, 24) == [2, 2]
+
+
+def test_check_human_age_when_animal_age_equal_27() -> None:
+    assert get_human_age(27, 27) == [2, 2]
+
+
+def test_check_human_age_when_cat_and_dog_age_equal_28() -> None:
+    assert get_human_age(28, 28) == [3, 2]
+
+
+def test_check_human_age_when_cat_and_dog_age_equal_100() -> None:
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
Rewrite your tests using `pytest` as mentioned in the task requirements. Besides that, include tests to check cases when incoming cat and dog years differ one from another or include negative numbers. Also test for raising exception when incorrect data types are provided as mentioned [in the checklist](https://github.com/MatsalakO/py-cat-and-dog-years/blob/develop/checklist.md#clean-code).